assets/src/components/account/UserProfile.tsx

```diff
@@ -189,6 +199,16 @@ class UserProfile extends React.Component<Props, State> {
               disabled={!isEditing}
             />
           </Box>
+          <Box mb={3} sx={{maxWidth: 480}}>
```

Comment:
Position is wrong as https://github.com/papercups-io/papercups/pull/160/files#diff-f310b3aeb13547d64314e5e0a19aa695R213 shows photo, let's move it after line 190.

Reply:
+1 that the position is wrong, also since we're not making the email field editable let's just put it at the top :)